Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -143,11 +143,18 @@ final class BuildPlanTests: XCTestCase {
             graph: graph,
             fileSystem: fileSystem))
 
-        XCTAssertEqual(Set(result.productMap.keys), ["APackageTests"])
+        XCTAssertEqual(Set(result.productMap.keys), ["BPackageTests", "APackageTests"])
       #if os(macOS)
-        XCTAssertEqual(Set(result.targetMap.keys), ["ATarget", "BTarget", "ATargetTests"])
+        XCTAssertEqual(Set(result.targetMap.keys), ["ATarget", "BTarget", "ATargetTests", "BTargetTests"])
       #else
-        XCTAssertEqual(Set(result.targetMap.keys), ["ATarget", "BTarget", "APackageTests", "ATargetTests"])
+        XCTAssertEqual(Set(result.targetMap.keys), [
+            "APackageTests",
+            "ATarget",
+            "BPackageTests",
+            "BTargetTests",
+            "ATargetTests",
+            "BTarget"
+        ])
```

Comment:
We should sort the keys for deterministic order

Reply:
Doesn't the conversion to a Set already handle that?

Reply:
Ah, right